configure.py

```diff
@@ -0,0 +1,63 @@
+import os, platform, subprocess
+
+def checkCommand(shell):
+    if platform.system() == "Windows":
+        return os.system("where " + shell)
```

Comment:
Prefer subprocess, with params passed as a list, e.g.:
subprocess.run(["ls", "-l"])

This removes problems with shell and is the standard way to run stuff from python.  You can also use variants such as subprocess.check_call() if that helps with error handling.

Reply:
Couldn't figure out how to get return codes from subprocess. os.system was easier.

Reply:
Resolving: changed to subprocess.check_call()